<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-OUBE3OH" failures="0" tests="1" name="Default test" time="69.457" errors="0" timestamp="14 Dec 2023 17:55:40 GMT">
  <testcase classname="org.testing.TestScripts.Final_TC1" name="@AfterMethod TestCase_TC1_Logout" time="0.0">
    <failure type="java.lang.NullPointerException" message="Cannot invoke &amp;quot;String.equals(Object)&amp;quot; because &amp;quot;actual&amp;quot; is null">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "String.equals(Object)" because "actual" is null
	at org.testing.assertion.Assertt.assertion1(Assertt.java:10)
	at org.testing.TestScripts.Final_TC1.TestCase_TC1_Logout(Final_TC1.java:62)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/java.lang.reflect.Method.invoke(Method.java:578)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:514)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:215)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:707)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- @AfterMethod TestCase_TC1_Logout -->
  <testcase classname="org.testing.TestScripts.Final_TC1" name="@AfterClass browser_close" time="0.0">
    <skipped/>
  </testcase> <!-- @AfterClass browser_close -->
  <testcase classname="org.testing.TestScripts.Final_TC1" name="Trending_Click" time="3.89"/>
</testsuite> <!-- Default test -->
